dockers/docker-orchagent/swssconfig.sh

```diff
@@ -22,11 +26,20 @@ function fast_reboot {
         swssconfig /default_routes.json
         mv -f /default_routes.json /default_routes.json.1
       fi
+  fi
+}
 
-      ;;
-    *)
-      ;;
-  esac
+function signal_kernel_restore {
```

Comment:
why is this only for config reload? can this be used for fast reboot as well?

Reply:
Fast reboot already restores the neighbor table to APP_DB, and to my knowledge does not need it restored to the kernel.